codalab/worker/worker.py

```diff
@@ -759,6 +772,27 @@ def bundle_service_reply(self, socket_id, err, message, data):
         else:
             self.bundle_service.reply(self.id, socket_id, message)
 
+    def save_time_stats(self, uuid, from_stage):
```

Comment:
It's a state, not a stage
```suggestion
    def save_time_stats(self, uuid, from_state):
```